pkg/controller/volume/persistentvolume/util/util.go

```diff
@@ -69,7 +69,7 @@ const (
 	// through the CSIMigration feature flags. When this annotation is set the
 	// Kubernetes components will "stand-down" and the external-provisioner will
 	// act on the objects
-	AnnMigratedTo = "volume.beta.kubernetes.io/migrated-to"
+	AnnMigratedTo = "kubernetes.io/migrated-to"
```

Comment:
I think we still want a "volume" prefix

Reply:
based on the other keys it looks like `pv` might actually be the appropriate prefix since it is applied to both `pv/pvc`. WDYT?

Reply:
looking at `AnnBoundByController` as a template

Reply:
We used `pv` slightly more than `volume` so that seems fine. I don't have a strong preference either way.